qiita_pet/support_files/config_portal.txt

```diff
@@ -0,0 +1,41 @@
+# ---------- Base information for the website ----------
+[sitebase]
+LOGO = /static/img/logo-clear.png
```

Comment:
Might be useful to note here (or in the docs in a later PR) what the size of this logo should be.